common/include/pcl/range_image/impl/range_image.hpp

```diff
@@ -101,39 +101,39 @@ RangeImage::createFromPointCloud (const PointCloudType& point_cloud, float angul
 }
 
 /////////////////////////////////////////////////////////////////////////
-template <typename PointCloudType> void 
+template <typename PointCloudType> void
 RangeImage::createFromPointCloud (const PointCloudType& point_cloud,
                                   float angular_resolution_x, float angular_resolution_y,
                                   float max_angle_width, float max_angle_height,
                                   const Eigen::Affine3f& sensor_pose, RangeImage::CoordinateFrame coordinate_frame,
                                   float noise_level, float min_range, int border_size)
 {
   setAngularResolution (angular_resolution_x, angular_resolution_y);
-  
+
   width  = static_cast<std::uint32_t> (pcl_lrint (std::floor (max_angle_width*angular_resolution_x_reciprocal_)));
   height = static_cast<std::uint32_t> (pcl_lrint (std::floor (max_angle_height*angular_resolution_y_reciprocal_)));
-  
+
   int full_width  = static_cast<int> (pcl_lrint (std::floor (pcl::deg2rad (360.0f)*angular_resolution_x_reciprocal_))),
       full_height = static_cast<int> (pcl_lrint (std::floor (pcl::deg2rad (180.0f)*angular_resolution_y_reciprocal_)));
   image_offset_x_ = (full_width -static_cast<int> (width) )/2;
   image_offset_y_ = (full_height-static_cast<int> (height))/2;
   is_dense = false;
-  
```

Comment:
Too many whitespace changes. Please split them into another commit. See [this link](https://til.hashrocket.com/posts/696df00135-remove-whitespace-changes-then-git-add-p) on how to create a code only commit from whitespace+code diff.

You'll need to rebase interactively, choose to edit a commit, reset the commit, apply the process mentioned above, then create 2 commits. WS changes are ok, but a 256 line diff searching for the removed variable adds too much noise